vividus-plugin-web-app/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     implementation project(':vividus-util')
     implementation project(':vividus-extension-selenium')
 
+    api project(':vividus-extension-selenium')
```

Comment:
why is this change needed?

Reply:
plugin-web-app based on extension-selenium but not exposes it as API.
For example, when I need to use WebJavaScriptActions  I need to add the extension in the dependencies because I'll get compile error otherwise